src/plugins/intel_cpu/tests/functional/single_layer_tests/reduce_ops.cpp

```diff
@@ -408,6 +413,19 @@ const auto params_MultiAxis_6D = testing::Combine(
         testing::Values(emptyCPUSpec),
         testing::Values(emptyFusingSpec));
 
+const auto params_Sum_Int32 = testing::Combine(
+        testing::Combine(
+            testing::ValuesIn(axes),
+            testing::Values(CommonTestUtils::OpType::VECTOR),
+            testing::ValuesIn(keepDims),
+            testing::Values(ngraph::helpers::ReductionType::Sum),
```

Comment:
Does it make sense to extend for other Reduction types? At least Min/Max would be valuable to cover as well.

Reply:
Good point! I've added Min/Max/L1, whose computation won't exceed the range of int32. Thanks Dmitry!